Support handle eBPF Access Logs and update related UI (#11732)
apache/skywalking: docs/en/concepts-and-designs/scope-definitions.md

| transmissionLatency | The latency from produce side to consume side .              |           | int(in ms) |
| status              | Indicates the success or failure of the request.             |           | boolean    |
| operation           | Indicates this access is on `Produce` or `Consume` side      |           | enum       |

### SCOPES with `K8S` Prefix

> All metrics starting with `K8S` are derived from monitoring the Access Log in Kubernetes.

Reviewer: ```suggestion
All metrics starting with `K8S` are derived from Kubernetes monitoring by Rover(eBPF agent).
```

#### Service, Service Instance and relations

> For all `K8SService`, `K8SInstance`, `K8SServiceRelation` and `K8SServiceInstanceRelation`, they all have the 

Reviewer: I think you missed those scopes' own field. Same for endpoint and endpoint relation.

Author: Updated.
